PRS8001-9000/PRS8788Severos.xml

```diff
@@ -39,21 +39,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="MKr" when="2020-06-06">Added external links</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
+            <person sex="1" sameAs="wd:Q2637658">
                <persName>
-                  
                   Severos of Ašmunayn
                </persName>
-               <persName type="alt">Sawiros zäʾǝsmunayn</persName>
-               <persName type="alt">Sawīros b. al-Muqaffaʿ</persName>
-               <occupation> rel. </occupation>
+               <persName xml:lang="gez" xml:id="n1" type="alt">Sāwiros zäʾǝsmunāyn</persName>
+               <persName xml:lang="gez" corresp="#n1" type="alt">ሳዊሮስ፡ ዘእስሙናይን፡</persName>
+               <persName xml:lang="ar" corresp="#n1" type="alt">Sawīros b. al-Muqaffaʿ</persName>
+               <persName xml:lang="ar" corresp="#n1" type="normalized">ساويرس بن المقفع</persName>
+               <occupation type="ecclesiastic">writer </occupation>
```

Comment:
sawiros bin al-muqaffa rather Needs another xml:id, and the Transliteration has the type normalized

Reply:
sorry, it should rather be like that
`<persName xml:lang="gez" xml:id="n1">ሳዊሮስ፡ ዘእስሙናይን፡ </persName>

               <persName xml:lang="gez" corresp="#n1"  type="normalized">Sāwiros za-ʾƎsmunāyn</persName>`

Reply:
Same for Arabic, so the Name in the original script has an id, and the transliterated form has a corresp